0.preprocessing_data/scripts/1.update_file_structure.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Copy raw images into one folder to use for CellProfiler processing
+#
+# Currently, the images are located nest deep within multiple folders.
+# For best practices, we will copy the images (preserving metadata) to one folder that can be used for CellProfiler processing.
+# This file is modified from its original version: https://github.com/WayScience/GFF_2D_organoid_prototyping .
```

Comment:
```suggestion

```
Probably can remove this or change it to the 3D repo as where it was modified from.